apps/mainsite/static/extensions/CompetencyExtension/context.json

```diff
@@ -0,0 +1,17 @@
+{
+  "@context": {
+    "obi": "https://w3id.org/openbadges#",
+    "extensions": "https://w3id.org/openbadges/extensions#",
+    "category": "extensions:category",
+    "escoID": "extensions:escoID",
+    "studyLoad": "extensions:studyLoad",
+    "name": "extensions:name",
+    "description": "extensions:description"
+  },
+  "obi:validation": [
+    {
+      "obi:validatesType": "extensions:CompetencyExtension",
+      "obi:validationSchema": "http://localhost:8000/static/extensions/CompetencyExtension/schema.json"
```

Comment:
This should also be the production url, even though it will only work once released

Reply:
adjusted the url